Assets/LeapMotion/Scripts/LeapImageRetriever.cs

```diff
@@ -89,6 +89,10 @@ public class LeapTextureData {
       TextureFormat format = getTextureFormat(image);
 
       if (_combinedTexture != null) {
+				if ((_combinedTexture.height == combinedHeight) &&
```

Comment:
Ok, I'll take this out. I just "stopped the bleeding", but I didn't know what was really going on.